spec/airbrake_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+RSpec.describe Airbrake do
+  describe "#add_rack_builder" do
+    it "adds new builder to the chain" do
+      builder = proc { |_, _| nil }
+      expect(Airbrake::Rack::NoticeBuilder).to receive(:add_builder) do |_, &block|
```

Comment:
Well, I think it's not good idea to really change the builder's list here because in this case it will impact all other specs.